test/integration/admin_views_an_individual_order_test.rb

```diff
@@ -23,12 +23,13 @@ class AdminViewsAnIndividualOrderTest < ActionDispatch::IntegrationTest
       click_link "Order number: #{order.id}"
     end
 
-    assert admin_order_path(order), current_path
 
-    assert page.has_content?("Order Date: #{date}")
+    assert order_path(order), current_path
 
-    assert page.has_content?("Purchaser's Name: #{order.user.username.capitalize}")
+    assert page.has_content?("Date of Order: #{order.created_at.strftime("%d %b. %Y")}")
```

Comment:
Line is too long. [88/80]<br>Prefer single-quoted strings inside interpolations.